### PR TITLE
adds prediction route to API

### PR DESCRIPTION
**Pull request by skhabiri**
new file:   Data/predict_df.csv
	modified:   project/app/api/dbpgsql.py
	new file:   project/app/api/gs_model
	modified:   project/app/api/predict.py
	modified:   project/app/main.py
	modified:   project/requirements.txt
	renamed:    requirements.txt -> requirements_old.txt	

deployed at : https://lab28dsk.bridgestoprosperity.dev/

### CODE REVIEW

File: project/app/api/dbpgsql.py [hunk 1]
```diff
@@ -29,51 +32,36 @@ def __init__(self):
...
+       'primary_crops_grown', 'river_crossing_deaths_in_last_3_years',
+       'river_crossing_injuries_in_last_3_years', 'incident_descriptions',
+       'notes_on_social_information', 'cell_service_quality',
+       'four_wd _accessibility', 'name_of_nearest_city',
+       'name_of_nearest_paved_or_sealed_road', 'bridge_classification',
+       'flag_for_rejection', 'rejection_reason', 'bridge_type',
+       'estimated_span_m', 'height_differential_between_banks',
+       'bridge_opportunity_general_project_photos',
+       'bridge_opportunity_casesafeid', 'country', 'good_site']
+
```
Comment (trevorwjames) [inline project/app/api/dbpgsql.py:58]: This second set is for the prediction table correct?

**Review by trevorwjames — APPROVED** [review]
Everything else looks good!

File: project/app/api/dbpgsql.py (hunk 1 above)
Comment (skhabiri) [reply project/app/api/dbpgsql.py:58]: yes, one table is cleaned main dataset. the second one is to get the query features for prediction purpose

Comment (trevorwjames) [reply project/app/api/dbpgsql.py:58]: Super! All Approved on my end.

Comment (skhabiri) [reply project/app/api/dbpgsql.py:58]: The model is housed in b2p-d.ipynb notebook, named gs_model.best_estimator_ and serialized as gs_model.

Comment (medamer) [reply project/app/api/dbpgsql.py:58]: Sounds good

**Comment by JonRivera** [comment]
Good work!

**Pull request merged**